html/controls/sk/sk-sheet.js

```diff
@@ -139,11 +127,11 @@ function prepareSkSheetTable(element, teamId, mode) {
 				}
 				row.find('.Trip2').text(scoreText);
 				otherRow.find('.Trip2').text('');
-			});
-			$.each(new Array(7), function (idx) {
-				var t = idx + 3;
-				WS.Register([prefix+'ScoringTrip('+t+').Score', prefix+'ScoringTrip('+t+').Current', 
-						prefix+'ScoringTrip('+t+').AfterSP'], function (k, v) {
+				break;
+
+			 default:
+				if (k.parts[4] == 'ScoringTrip' && k.ScoringTrip >= 3 && k.ScoringTrip <= 10) {
```

Comment:
This should be <10.
Trip 10 is already part of the cell that's treated below.

Reply:
Changed